Add improvements to concept docs. (#2655)
wso2/docs-is: en/docs/references/concepts/authorization/access-token.md

# Introduction to Access Token

## Access token

Access tokens are credentials used to access protected resources on behalf of a resource owner. These tokens can be 
either an opaque string or a JSON Web Token (JWT). The **resource server**<!--[**Resource Server**](insertlink)--> can decide whether a client is authorized to 
consume a resource or not, by verifying the access token. The way the token endpoint of the **authorization server**<!--[**Authorization Server**](insertlink)--> is 
invoked and the access token is generated differs based on the **grant type**<!--[grant type](insertlink)-->. The ultimate audience of the access 

> token is the resource server, so the client can not interpret the value of the access token.

Reviewer: ```suggestion
token is the resource server, so the client cannot interpret the value of the access token.
```

![The flow to obtain an access token](../../../assets/img/concepts/access-token.png)

1. The client sends a request for an access token to the authorization server by passing the required parameters of the 
   relevant grant type.
2. The authorization server validates the parameters and grants an access token to the client.
3. The client passes the access token to the authorization server.

> 4. Based on the validity of the access token the resource server allows the client to consume the protected resource.

Reviewer: ```suggestion
4. Based on the validity of the access token, the resource server allows the client to consume the protected resource.
```
